src/script/components/Meeting/MeetingList/MeetingList.tsx

```diff
@@ -126,10 +83,14 @@ export const MeetingList = () => {
   }
 
   if (isNextTab) {
```

Comment:
Instead of next I would rather use the word upcoming. Next is often used to describe an action, but we are looking for upcoming meetings